pkg/monitor/monitor/util_test.go

```diff
@@ -51,6 +51,8 @@ func TestGetMonitorService(t *testing.T) {
 							"app.kubernetes.io/managed-by": "tidb-operator",
 							"app.kubernetes.io/instance":   "foo",
 							"app.kubernetes.io/component":  "monitor",
+							"app.kubernetes.io/used-by":    "prometheus",
+							"kubernetes.io/name":           "foo-prometheus",
```

Comment:
I find this test haven't covered the grafana. Could you extend this unit-test to cover grafana?

Reply:
of course, done